app.rb

```diff
@@ -1,8 +1,25 @@
 require 'sinatra/base'
+require './lib/player.rb'
 class RockPaperScissors < Sinatra::Base
   get '/test' do
     'test page'
   end
 
   run! if app_file == $0
+
+  get '/names' do
```

Comment:
Your application starts on `/names`. A suggestion may be for `/names` to be replaced with `/` so your application can run from `localhost:9292` without explicitly needing to redirect to `/names`.  Currently, when I run your app in the browser, it returned a 404 in the terminal due to this.